appveyor/Convert-FileUrisToGitHubUris.ps1

```diff
@@ -0,0 +1,120 @@
+$location=$args[0]
+$repoProvider=$env:APPVEYOR_REPO_PROVIDER
+$projectSlug=$env:APPVEYOR_PROJECT_SLUG
+$repoName=$env:APPVEYOR_REPO_NAME
+$repoCommit=$env:APPVEYOR_REPO_COMMIT
+#$repoProvider='gitHub'
+#$projectSlug='sarif-sdk'
+##$projectSlug='sarif-sdk-931m5'
+#$repoName='Rolstenhouse/sarif-sdk'
+#$repoCommit='f82ebd892917bd6157efc04b832a57964ec1f2c6'
+
+$githubBaseUri = ""
+
+if($repoProvider -eq "gitHub")
+{
+  $githubBaseUri = "https://raw.githubusercontent.com/"
+}
```

Comment:
If the provider isn't `github`, shouldn't we just error out? Otherwise, `Rebase-Uri` returns a string like `Rolstenhouse/sarif-sdk/f82...2c6/..." with no host name.